providers/song-info-front.js

```diff
@@ -10,17 +10,16 @@ ipcRenderer.on("update-song-info", async (_, extractedSongInfo) => {
 });
 
 const injectListener = () => {
-	var oldXHR = window.XMLHttpRequest;
+	const oldXHR = window.XMLHttpRequest;
 	function newXHR() {
-		var realXHR = new oldXHR();
+		const realXHR = new oldXHR();
 		realXHR.addEventListener(
 			"readystatechange",
 			() => {
-				if (realXHR.readyState == 4 && realXHR.status == 200) {
-					if (realXHR.responseURL.includes("/player")) {
+				if (realXHR.readyState === 4 && realXHR.status === 200
+					&& realXHR.responseURL.includes("/player")) {
 						// if the request contains the song info, send the response to ipcMain
 						ipcRenderer.send("song-info-request", realXHR.responseText);
-					}
```

Comment:
this is a pretty random lint but xo was warning me about those so i decided why not